### PR TITLE
styles: adds scrollTo projectsContainer, mobile styles

### PR DESCRIPTION
**Pull request by jedwardmook**
- adds scrollTo to projectsContainer, clicking on project will scroll up to heroProject
- nests media queries
- changes font-sizes from vws to rems

### CODE REVIEW

File: src/components/ProjectsContainer.js [hunk 1]
```diff
@@ -19,6 +19,10 @@ function ProjectsContainer({styles, projects, setHeroProject}) {
     }
   };
 
+  const handleHeroProject = (project) => {
```
Comment (inkbeard) [inline src/components/ProjectsContainer.js]: Why is this needed? it's a single method that calls another single method.

File: src/components/projects.module.css [hunk 2]
```diff
@@ -181,12 +258,20 @@
 }
 
 .image-viewer-hero-description {
-  font-size: 1.75vw;
+  font-size: 1.15rem;
```
Comment (inkbeard) [inline src/components/projects.module.css]: I would reverse this so it's a consistent "mobile-first" approach with:

```
.image-viewer-hero-description {
  font-size: .75rem;
  font-family: var(--neue-family);
  
  @media (min-width: 600px) {
    font-size: 1rem;
  }
    
  @media (min-width: 800px) {
    font-size: 1.15rem;
  }
}
```

**Review by inkbeard — COMMENTED** [review]
Great job with the rearranging of the font-sizes and the media queries, but I would do 2 things here. 

1. I would change `@media (width <= ###px) {` to be `@media (min-width: ###px) {`. I've seen this more than using `<=` and `>=`, which is evidently called "inclusive" according to https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_media_queries/Using_media_queries. Personally, I find it more human readable, but really, if you're on a team where they've decided to do it one or the other, some times you just gotta do it to be consistent.
2. This isn't really a "mobile-first" approach. You're basically do the opposite (non-mobile-first?). The idea is that you don't want to override things unless you need to. https://www.springboard.com/blog/design/what-is-mobile-first-design/ has some good info on here too.

I provided an example for these 2 suggestions as well.

File: src/components/projects.module.css (hunk 2 above)
Comment (jedwardmook) [reply src/components/projects.module.css]: Thanks! This makes sense.

File: src/components/ProjectsContainer.js (hunk 1 above)
Comment (jedwardmook) [reply src/components/ProjectsContainer.js]: There are two reasons, both no longer make sense in the way it's set up now. It's a bit superfluous. The plan was to handle the scrollTo in the function, but adding an anchor tag proved easier/semantic. The other was I had heard that having inline functions on an onClick can add extra rerenders, but I am still doing that so I might as well, not have an inline function that calls another function when I can just call setHeroProject.

File: src/components/projects.module.css (hunk 2 above)
Comment (jedwardmook) [reply src/components/projects.module.css]: I do get this error when I try this method: 'Expected "context" media feature range notation (media-feature-range-notation)'

Comment (inkbeard) [reply src/components/projects.module.css]: Ah, it looks like that's coming from https://stylelint.io/user-guide/rules/media-feature-range-notation/. You would need to change it from "context" to "prefix" I guess. It says it can be "limiting" due to https://drafts.csswg.org/mediaqueries/#mq-min-max, but this sort of seems sort of edge case to me. I'm good with way as it's you're project as long as you do mobile first.

**Review by inkbeard — APPROVED** [review]
